docs/coordinates/index.rst

```diff
@@ -1,13 +1,12 @@
-.. We call EarthLocation.of_site here first to force the downloading
-.. of sites.json so that future doctest output isn't cluttered with
-.. "Downloading ... [done]". This can be removed once we have a better
-.. way of ignoring output lines based on pattern-matching, e.g.:
+.. Force downloading of sites.json so that future doctest output isn't
+.. cluttered with "Downloading ... [done]". This can be removed once we have a
+.. better way of ignoring output lines based on pattern-matching, e.g.:
```

Comment:
I'm actually confused about this comment: it seems like in the existing documentation, `of_site()` isn't actually called below (that doctest on L339 is skipped?). But I think that might be ok: given the issues we have with remote data tests and the stability of the astropy-data server, we might consider keeping it as is and removing this setup instead so that this page doesn't actually make any additional remote data calls. Thoughts? (and cc @pllim)

Reply:
Is this setup not needed anywhere else in the remote data doctest suite?

I'd rather you use the local data file but I don't know the *intent* of this test.

Reply:
My vote is: Less remote data the better, if possible.

Reply:
> Is this setup not needed anywhere else in the remote data doctest suite?

It looks like most calls to `of_site()` in the docs are skipped, but there are a few that aren't, so we can address this aspect in a separate PR/issue.

Reply:
I agree with @eerovaher's point that this caught a real genuine bug, so while I agree with "less remote data the better, if possible", I think in this case it's really "minimal necessary to catch bugs", and this is catching bugs!